src/content/docs/style-guide/writing-guidelines/levels-headings.mdx

```diff
@@ -35,11 +35,11 @@ If your doc's title is long and you would like a shorter title to appear in the
 Organize chunks of information into sections with level two headings (`##`). For example:
 
 ```
-## Create a new user
+## Create a new user  [#create-new-user]
 ```
 
 <Callout variant="important">
-  Our site will use your header text as that header's ID.
+  If you don't specify an ID manually, the site will use your header text as that header's ID. Creating a manual ID can help preserve anchor links when docs move around and get rewritten.
```

Comment:
[nit] The manual `ID` (or custom`ID`, whichever term you like 😜 ), isn't so much about preserving the location when a doc moves, but rather if the title changes. Without the custom ID, the site will create a derived ID by using the text in the heading. So for example "Ruby agent examples", would get an automatically generated ID of `ruby-agent-examples`. If this heading were to change to `The world's best ruby agent examples`, then the auto ID would get generated as `the-worlds-best-ruby-agent-examples`, and you'd lose reference to the old one.

Moving a page doesn't have any affect on the ID itself.